test/crd.go

```diff
@@ -197,7 +197,7 @@ func EventSenderPod(name string, namespace string, sink string, event CloudEvent
 			Containers: []corev1.Container{{
 				Name:            "sendevent",
 				Image:           ImagePath("sendevent"),
-				ImagePullPolicy: corev1.PullAlways, // TODO: this might not be wanted for local.
+				ImagePullPolicy: corev1.PullIfNotPresent, // needed for local testing
```

Comment:
I find this to not work in a real cluster if I run two tests back to back. PullAlways forces a repull of an image that is more up to date.

Reply:
@mattmoor was saying this is not going to work well because of how the registry works in clouds. So this will break the e2e test in gke.

Reply:
I think the right answer is you need to wire in a flag to crd.go to use one or the other based on the env flag.

Reply:
@n3wscott can you check now please? The e2e tests passed. Thanks!